apps/interactiveevents/templatetags/react_interactiveevents.py

```diff
@@ -27,7 +27,7 @@ def react_interactiveevents(context, obj):
 
     private_policy_label = str(_('I confirm that I have read and accepted the '
                                  '{}terms of use{} and the {}data protection '
-                                 'policy{}.'))
+                                 'policy{}.*'))
```

Comment:
Can we move it out of the tranlated string? Or is that too annoying with our special react-link-thing? Then, we should leave it for next time, when we get new trsnalations. Sorry!

Reply:
Haha, translation is a very hard word